libs/ui/elements/src/lib/downloads-list/downloads-list.component.spec.ts

```diff
@@ -132,9 +132,6 @@ describe('DownloadsListComponent', () => {
         LINK_FIXTURES.geodataShpWithMimeType
       )
       expect(items[0].componentInstance.format).toEqual('shp')
-      expect(items[0].componentInstance.color).toEqual(
```

Comment:
Is this change necessary in this PR? I'm just wondering why this got removed.

Reply:
It is because the color getter is deleted. So we can't call it anymore :  https://github.com/geonetwork/geonetwork-ui/pull/747/files#diff-5cfed937e30ec743c98e5e024350945ee6d4d0b6e7e34dabac9f3fb580fa0ad4L27-L28

Reply:
that shouldn't be impacted here, unless I'm missing something; in general we should strive to avoid reducing the test coverage :)

Reply:
Oh yeah indeed, I don't know why my IDE displayed it in red. Sry @Angi-Kinas for the misunderstood !